app/views/includes/forms/card.scala.html

```diff
@@ -37,7 +39,16 @@
         @heading
       }
     </@headingTag>
-    <p> @bodyContent</p>
+    @if(isBodyContentList){
```

Comment:
not sure about this so @sergi-hmrc correct me if wrong but would doing 
@if(bodyContentList.nonEmpty)
be better as then we wouldnt have to add extra parameters

Reply:
isn't the intention to always have a list? then why do we keep a list and a body content? 
Shouldn't it just be `bodyContentList: Seq[String] = Seq.empty`

Reply:
for now, and as @brendanstraughan said, just use nonEmpty. no need to add another parameter

Reply:
cool updated